openbas-api/src/test/java/io/openbas/utils/fixtures/ExpectationFixture.java

```diff
@@ -1,62 +1,55 @@
 package io.openbas.utils.fixtures;
 
-import io.openbas.database.model.Asset;
-import io.openbas.database.model.AssetGroup;
+import io.openbas.database.model.*;
 import io.openbas.model.expectation.DetectionExpectation;
 import io.openbas.model.expectation.PreventionExpectation;
 import io.openbas.rest.exercise.form.ExpectationUpdateInput;
-import java.util.Collections;
+import java.util.List;
 
 public class ExpectationFixture {
 
   static Double SCORE = 100.0;
 
+  public static PreventionExpectation createTechnicalPreventionExpectation(
```

Comment:
Can you give a more explicit word for the method please ? It's unclear if we are creating an expectation for an asset or for an agent.